### PR TITLE
Teamsfx Samples Migration

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

File: samples/graph-bulk-meetings/nodejs/teamsapp.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: 1.0.0
+
+environmentFolderPath: ./env
+projectId: 569f0fc9-71e8-49fd-a954-4afa10012c40
```
Comment (formulahendry) [inline samples/graph-bulk-meetings/nodejs/teamsapp.yml:6]: remove projectId

File: samples/graph-bulk-meetings/nodejs/client/appPackage/manifest.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
```
Comment (formulahendry) [inline samples/graph-bulk-meetings/nodejs/client/appPackage/manifest.json:1]: we have two manifest in this sample?

File: samples/graph-bulk-meetings/nodejs/teamsapp.local.yml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+      appPackagePath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip
+    # Write the information of created resources into environment file for
+    # the specified environment variable(s).
+    writeToEnvironmentFile:
+      titleId: M365_TITLE_ID
+      appId: M365_APP_ID
+
+deploy:
+  # Run npm command
+  - uses: cli/runNpmCommand
```
Comment (formulahendry) [inline samples/graph-bulk-meetings/nodejs/teamsapp.local.yml:93]: npm install for client as well?

File: samples/graph-bulk-meetings/nodejs/infra/azure.bicep
Comment (formulahendry) [inline samples/graph-bulk-meetings/nodejs/infra/azure.bicep:1]: Is infra folder needed?

File: samples/graph-bulk-meetings/nodejs/server.js
Comment (formulahendry) [inline samples/graph-bulk-meetings/nodejs/server.js:1]: have we updated readme?

Comment (Mohammed-MSFT) [reply samples/graph-bulk-meetings/nodejs/server.js:1]: fixed

File: samples/graph-bulk-meetings/nodejs/teamsapp.yml (hunk 1 above)
Comment (Mohammed-MSFT) [reply samples/graph-bulk-meetings/nodejs/teamsapp.yml:6]: fixed

File: samples/graph-bulk-meetings/nodejs/client/appPackage/manifest.json (hunk 2 above)
Comment (Mohammed-MSFT) [reply samples/graph-bulk-meetings/nodejs/client/appPackage/manifest.json:1]: fixed

File: samples/graph-bulk-meetings/nodejs/teamsapp.local.yml (hunk 3 above)
Comment (Mohammed-MSFT) [reply samples/graph-bulk-meetings/nodejs/teamsapp.local.yml:93]: fixed

File: samples/graph-bulk-meetings/nodejs/infra/azure.bicep
Comment (Mohammed-MSFT) [reply samples/graph-bulk-meetings/nodejs/infra/azure.bicep:1]: fixed

**Review by formulahendry — APPROVED** [review]

**Review by Prithvi-MSFT — APPROVED** [review]

**Review by Prithvi-MSFT — CHANGES_REQUESTED** [review]
Run build pipeline

File: samples/graph-bulk-meetings/nodejs/infra/azure.bicep
Comment (Mohammed-MSFT) [reply samples/graph-bulk-meetings/nodejs/infra/azure.bicep:1]: fixed

**Comment by Mohammed-MSFT** [comment]
Executed Build Pipeline :
 [Teamsfx Samples Migration by Mohammed-MSFT · Pull Request #886 · OfficeDev/Microsoft-Teams-Samples (github.com)](https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/886)

**Comment by ChetanSharma-msft** [comment]
@Mohammed-MSFT - Kindly resolve merge conflicts and resolve all the fixed comments before merging this PR.

**Review by Prithvi-MSFT — APPROVED** [review]

**Comment by Mohammed-MSFT** [comment]
**Executed Build PileLine :** https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5876851979

File: samples/app-anonymous-users/nodejs/ClientApp/package.json [hunk 4]
```diff
@@ -13,10 +13,11 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.4.3",
-    "react-scripts": "^5.0.1",
+    "react-scripts": "^2.1.3",
```
Comment (ChetanSharma-msft) [inline samples/app-anonymous-users/nodejs/ClientApp/package.json:16]: Why we have reverted to use older version?

File: samples/app-anonymous-users/nodejs/ClientApp/src/index.js [hunk 5]
```diff
@@ -4,13 +4,15 @@
 // </copyright>
 
 import { FluentProvider, teamsLightTheme } from '@fluentui/react-components';
-import React from 'react';
-import ReactDOM from 'react-dom';
+import React from "react";
+// import ReactDOM from "react-dom";
```
Comment (ChetanSharma-msft) [inline samples/app-anonymous-users/nodejs/ClientApp/src/index.js:8]: Remove commented code

File: samples/app-anonymous-users/nodejs/aad.manifest.json [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+            "isEnabled": true,
+            "type": "User",
+            "userConsentDescription": "Enable Teams to call this app's web APIs with the same rights that you have",
+            "userConsentDisplayName": "Teams can access app's web APIs and make requests on your behalf",
+            "value": "access_as_user"
+        }
+    ],
+    "preAuthorizedApplications": [
+        {
+            "appId": "1fec8e78-bce4-4aaf-ab1b-5451cc387264",
```
Comment (ChetanSharma-msft) [inline samples/app-anonymous-users/nodejs/aad.manifest.json:46]: Why are you pushing app id?

File: samples/app-anonymous-users/nodejs/aad.manifest.json [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+                "${{AAD_APP_ACCESS_AS_USER_PERMISSION_ID}}"
+            ]
+        },
+        {
+            "appId": "5e3ce6c0-2b1f-4285-8d4b-75ee78787346",
+            "permissionIds": [
+                "${{AAD_APP_ACCESS_AS_USER_PERMISSION_ID}}"
+            ]
+        },
+        {
```
Comment (ChetanSharma-msft) [inline samples/app-anonymous-users/nodejs/aad.manifest.json:57]: What are these app id's and why we are pushing?

File: samples/app-anonymous-users/nodejs/aad.manifest.json [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+    "identifierUris":[
+        "api://botid-${{AAD_APP_CLIENT_ID}}"
+    ],
+    "replyUrlsWithType":[
+        {
+          "url": "https://${{BOT_DOMAIN}}/auth-end.html",
+          "type": "Web"
+        },
+        {
+			"url": "https://token.botframework.com/.auth/web/redirect",
```
Comment (ChetanSharma-msft) [inline samples/app-anonymous-users/nodejs/aad.manifest.json:103]: Check alignment

File: samples/app-anonymous-users/nodejs/appPackage/manifest.json [hunk 9]
```diff
@@ -76,11 +76,11 @@
     "messageTeamMembers"
   ],
   "validDomains": [
-    "{{domain-name}}"
+    "${{BOT_DOMAIN}}"
   ],
   "webApplicationInfo": {
-    "id": "{{Microsoft-App-id}}",
-    "resource": "api://{{domain-name}}/botid-{{Microsoft-App-id}}"
+    "id": "${{AAD_APP_CLIENT_ID}}",
```
Comment (ChetanSharma-msft) [inline samples/app-anonymous-users/nodejs/appPackage/manifest.json:82]: Make sure that these place holders are updated in Readme for running the sample local.

File: samples/graph-bulk-meetings/nodejs/aad.manifest.json [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+        },
+        {
+            "appId": "4765445b-32c6-49b0-83e6-1d93765276ca",
+            "permissionIds": [
+                "${{AAD_APP_ACCESS_AS_USER_PERMISSION_ID}}"
+            ]
+        },
+        {
+            "appId": "4345a7b9-9a63-4910-a426-35363201d503",
+            "permissionIds": [
```
Comment (ChetanSharma-msft) [inline samples/graph-bulk-meetings/nodejs/aad.manifest.json:103]: Do we need to push these app ids?

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

File: samples/app-anonymous-users/nodejs/ClientApp/package.json (hunk 4 above)
Comment (Mohammed-MSFT) [reply samples/app-anonymous-users/nodejs/ClientApp/package.json:16]: fixed

File: samples/app-anonymous-users/nodejs/ClientApp/src/index.js (hunk 5 above)
Comment (Mohammed-MSFT) [reply samples/app-anonymous-users/nodejs/ClientApp/src/index.js:8]: fixed

File: samples/app-anonymous-users/nodejs/aad.manifest.json (hunk 6 above)
Comment (Mohammed-MSFT) [reply samples/app-anonymous-users/nodejs/aad.manifest.json:46]: auto generated, updated to empty string

File: samples/graph-bulk-meetings/nodejs/aad.manifest.json (hunk 10 above)
Comment (Mohammed-MSFT) [reply samples/graph-bulk-meetings/nodejs/aad.manifest.json:103]: Yes we need to push this file as it is.
Please refer this PR Once : https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/913/files

File: samples/app-anonymous-users/nodejs/aad.manifest.json (hunk 8 above)
Comment (Mohammed-MSFT) [reply samples/app-anonymous-users/nodejs/aad.manifest.json:103]: alignment fixed.

File: samples/app-anonymous-users/nodejs/aad.manifest.json (hunk 7 above)
Comment (Mohammed-MSFT) [reply samples/app-anonymous-users/nodejs/aad.manifest.json:57]: we need to push this file

File: samples/app-anonymous-users/nodejs/appPackage/manifest.json (hunk 9 above)
Comment (Mohammed-MSFT) [reply samples/app-anonymous-users/nodejs/appPackage/manifest.json:82]: yes it is.

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine.

**Comment by ChetanSharma-msft** [comment]
Executed build pipeline: https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5949363667

**Comment by ChetanSharma-msft** [comment]
image

**Pull request merged**